sandy/core/endf6.py

```diff
@@ -87,8 +88,9 @@ def get_endf6_file(library, kind, zam, to_file=False):
             * 'nfpy' is a Neutron-Induced Fission Product Yields nuclear data
               file
             * 'decay' is a Radioactive Decay Data nuclear data file
-    zam : `int` or 'all'
-        zam = 'int' (individual nuclides)
+    zam : `int` or `list` or 'all'
+        zam = 'int' (individual nuclides) or `list` or `tuple` or 
+        `numpy.ndarray` (group of nuclides)
```

Comment:
And what about a pandas.Series, or a Map object?
You should include all possible types of iterable...then the only way is to check if they have dunder `__len__`